CIP-0100/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CIP: 100?
```

Comment:
```suggestion
CIP: ????
```
CIP editors will apply the number during this CIP's acknowledgement in a CIPs editor meeting.

Reply:
Currently before candidacy we ask for a single `?` in the header and assign `????` in the PR title 🤓 https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#header-preamble